draft-bbs-signatures.md

```diff
@@ -426,51 +424,41 @@ Procedure:
 
 5. if KeyValidate(PK) is INVALID abort
 
-6. b = P1 + H0 * s + H_1 * msg_1 + ... + H_L * msg_L
-
-7. r1 = HASH(PRF(8*ceil(log2(q)))) mod q
-
-8. r2 = HASH(PRF(8*ceil(log2(q)))) mod q
-
-9. e~ = HASH(PRF(8*ceil(log2(q)))) mod q
-
-10. r2~ = HASH(PRF(8*ceil(log2(q)))) mod q
-
-11. r3~ = HASH(PRF(8*ceil(log2(q)))) mod q
+6. for rand_el in (r1, r2, e~, r2~, r3~, s~): rand_el = HASH(PRF(8*ceil(log2(q)))) mod q
```

Comment:
Can you please add a check for rand_el = 0 like above

Reply:
Done! Thank you!